presto-docs/src/main/sphinx/optimizer/reorder-joins.rst

```diff
@@ -63,8 +65,19 @@ For the following query:
 
 .. code-block:: sql
 
-  SELECT * FROM part p, orders o, lineitem l, supplier s, nation n
-  WHERE p.partkey = l.partkey AND l.orderkey = o.orderkey AND l.suppkey = s.suppkey AND s.nationkey = n.nationkey;
+ SELECT * FROM part p, orders o, lineitem l, supplier s, nation n
+ WHERE p.partkey = l.partkey AND l.orderkey = o.orderkey AND l.suppkey = s.suppkey AND s.nationkey = n.nationkey;
 
 The join order will change from ``part, orders, lineitem, supplier, nation`` to
 ``part, lineitem, orders, supplier, nation``.
+
+
+COST_BASED
+==========
+
+When the join reordering strategy is set to ``COST_BASED``, Presto will use statistics
```

Comment:
Let's link to a section of docs describing Stats in Presto. How to gather them, display etc.